ansible/vars/dev/dev_public.yml

```diff
@@ -89,10 +89,23 @@ backup_es: False
 postgres_users:
   commcare:
     username: "{{ localsettings.PG_DATABASE_USER }}"
-    password: "{{ secrets.pg_user_passwords.default }}"
+    password: commcarehq
   devreadonly:
     username: "devreadonly"
-    password: "{{ secrets.pg_user_passwords.devreadonly }}"
+    password: devreadonly
+    role_attr_flags: 'NOSUPERUSER,NOCREATEROLE,NOCREATEDB'
+    privs:
+      -
+        privs: SELECT
+        objs: ALL_IN_SCHEMA
+  replication:
+      username: 'hqrepl'
+      password: 'hqrepl'
+      role_attr_flags: 'LOGIN,REPLICATION'
+
+pg_replication_slots:
+ 192.168.33.16:
+   - standby1
```

Comment:
Since this is per host it could be set in the inventory or in a group vars file for each host.